CHANGELOG.next.asciidoc

```diff
@@ -413,6 +413,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add fileset to the Zeek module for the intel.log. {pull}14404[14404]
 - Add vpc flow log fileset to AWS module. {issue}13880[13880] {pull}14345[14345]
 - Add document for Filebeat input httpjson. {pull}14602[14602]
+- Fix timezone parsing in haproxy pipeline. {pull}14755[14755]
```

Comment:
nit: shouldn't that be updated to this PR?

Reply:
The PR number should keep the same so it can be tracked back to the original PR 😄